src/main/java/loqor/ait/core/engine/impl/LifeSupportCircuit.java

```diff
@@ -0,0 +1,70 @@
+package loqor.ait.core.engine.impl;
+
+import java.util.List;
+
+import net.minecraft.block.Blocks;
+import net.minecraft.entity.LivingEntity;
+import net.minecraft.util.math.BlockPos;
+import net.minecraft.util.math.Direction;
+
+import loqor.ait.core.AITBlocks;
+import loqor.ait.core.engine.DurableSubSystem;
+import loqor.ait.core.engine.StructureHolder;
+import loqor.ait.core.engine.block.multi.MultiBlockStructure;
+import loqor.ait.core.tardis.util.TardisUtil;
+
+public class LifeSupportCircuit extends DurableSubSystem implements StructureHolder {
```

Comment:
where is the block for this? you will need to create a block and a block entity which extends `SubSystemBlockEntity`
Look at `EngineBlockEntity` etc for help

Reply:
i was wondering whether i had to or not